.circleci/generate_circleci_config.py

```diff
@@ -13,18 +14,34 @@
 # https://hub.docker.com/r/kindest/node/tags
 # This should match what is in tests/__init__.py
 KUBE_VERSIONS = ["1.19.11", "1.20.7", "1.21.2", "1.22.7", "1.23.4"]
+CI_REMOTE_DOCKER_VERSION = "20.10.12"
+
+
+def list_docker_images(path):
+    command = "cd " + path + " && make show-docker-images-no-link"
```

Comment:
This is probably more legible with f-strings
```suggestion
    command = f"cd {path} && make show-docker-images-no-link"
```